scss/_tooltip.scss

```diff
@@ -37,21 +42,34 @@
     width: var(--#{$prefix}tooltip-arrow-width);
     height: var(--#{$prefix}tooltip-arrow-height);
 
-    &::before {
+    &::before,
+    &::after { // Boosted mod
       position: absolute;
+      display: block; // Boosted mod
       content: "";
       border-color: transparent;
       border-style: solid;
+      border-width: 0; // Boosted mod
     }
   }
 }
 
+// Boosted mod: new design for tooltips with a different color for borders
```

Comment:
We need to remove the notion of "new design" in this comment since it's new right now but not gonna be new forever